_components/search-bar.md

```diff
@@ -77,6 +77,9 @@ title: Search Bar
       <li><a href="{{ site.baseurl }}/components/#forms-blocks">accessibility guidelines for form templates</a> and</li>
       <li><a href="{{ site.baseurl }}/elements/#inputs">the accessibility guidelines for form controls</a>.</li>
     </ul>
-    <p>There are no other specific accessibility guidelines for this form template.</p>
+    <p>We also recommend:</p>
+    <ul>
+      <li>Even when the search button contains an icon and no text, make sure the word "search" is still inside the <button> element. You can hide this text by absolutely positioning it to -999em (as we have here). This descriptive text tells screen reader users the button's purpose.
```

Comment:
Agreed.

@colinpmacarthur we also want to be sure to add `<code>` around parts of the copy that are related to coding examples.